frontend/src/common/Utils.ts

```diff
@@ -38,4 +38,31 @@ export class Utils {
             `https://rinkeby.etherscan.io/tx/${tid}` :
             `https://etherscan.io/tx/${tid}`;
     }
+
+    static validateMultipleEmails (emailInput: any) {
+        // Get value on emails input as a string
+        var emails = emailInput;
```

Comment:
Dont use `var` user `const` if absolutely necessary the change the value then use `let`